### PR TITLE
Updated cartridge documentation

### PR DESCRIPTION
**Pull request by msteijaert**
Made examples compatible with latest chembl (25) and most recent conda versions of rdkit (2019.03.4.0, python 3.6.9) + rdkit-postgresql (2019.03.4.0)

### CODE REVIEW

File: Docs/Book/Cartridge.md [hunk 1]
```diff
@@ -65,29 +65,29 @@ Start by downloading and installing the postgresql dump from the ChEMBL website
...
-    chembl_23=# create extension if not exists rdkit;
-    chembl_23=# create schema rdk;
+    chembl_25=# create extension if not exists rdkit;
+    chembl_25=# create schema rdk;
 
 Create the molecules and build the substructure search index:
 
-    chembl_23=# select * into rdk.mols from (select molregno,mol_from_ctab(molfile::cstring) m  from compound_structures) tmp where m is not null;
+    chembl_25=# select * into rdk.mols from (select molregno,mol_from_ctab(molfile::cstring) m  from compound_structures) tmp where m is not null;
     SELECT 1727081
```
Comment (greglandrum) [inline Docs/Book/Cartridge.md]: I think this line should probably also have changed when you updated to ChEMBL25

File: Docs/Book/Cartridge.md [hunk 2]
```diff
@@ -65,29 +65,29 @@ Start by downloading and installing the postgresql dump from the ChEMBL website
...
     CREATE INDEX
-    chembl_23=# alter table rdk.mols add primary key (molregno);
+    chembl_25=# alter table rdk.mols add primary key (molregno);
     ALTER TABLE
 
 Create some fingerprints and build the similarity search index:
 
-    chembl_23=# select molregno,torsionbv_fp(m) as torsionbv,morganbv_fp(m) as mfp2,featmorganbv_fp(m) as ffp2 into rdk.fps from rdk.mols;
+    chembl_25=# select molregno,torsionbv_fp(m) as torsionbv,morganbv_fp(m) as mfp2,featmorganbv_fp(m) as ffp2 into rdk.fps from rdk.mols;
     SELECT 1727081
```
Comment (greglandrum) [inline Docs/Book/Cartridge.md]: Same comment here.

File: Docs/Book/Cartridge.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ Given we're searching through 1.7 million compounds these search times aren't in
 
 One easy way to speed things up, particularly for queries that return a large number of results, is to only retrieve a limited number of results:
 
-    chembl_23=# select * from rdk.mols where m@>'c1cccc2c1CNCCN2' limit 100;
+    chembl_25=# select * from rdk.mols where m@>'c1cccc2c1CNCCN2' limit 100;
      molregno |                                                                                             m                                                             
 
     ----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (greglandrum) [inline Docs/Book/Cartridge.md]: Don't the results returned from this query (and the next one) change as well?

File: Docs/Book/Cartridge.md (hunk 1 above)
Comment (msteijaert) [reply Docs/Book/Cartridge.md]: updated

File: Docs/Book/Cartridge.md (hunk 2 above)
Comment (msteijaert) [reply Docs/Book/Cartridge.md]: updated

File: Docs/Book/Cartridge.md (hunk 3 above)
Comment (msteijaert) [reply Docs/Book/Cartridge.md]: updated

**Comment by greglandrum** [comment]
Thanks!

**Pull request merged**